packages/database/src/schema.prisma

```diff
@@ -444,10 +481,13 @@ model Event {
 model NeighbouringAddress {
   id String @id @default(dbgenerated("newid()")) @db.UniqueIdentifier
 
-  addressLine1 String
-  addressLine2 String?
-  townCity     String
-  postcode     String
+  addressLine1      String
+  addressLine2      String?
+  townCity          String
+  county            String?
+  postcode          String
+  siteAccessDetails String? @db.NVarChar(MAX) /// json array of information on site accessibility
+  siteSafetyDetails String? @db.NVarChar(MAX) /// json array of information on site health and safety 
 
   // relations
   caseReference String
```

Comment:
should we attempt to normalise addresses, currently just linked to case directly?